library/Mockery/Mock.php

```diff
@@ -279,7 +279,7 @@ public function expects($something = null)
     /**
      * Shortcut method for setting an expectation that a method should not be called.
      *
-     * @param array ...$methodNames one or many methods that are expected not to be called in this mock
+     * @param mixed ...$methodNames one or many methods that are expected not to be called in this mock
```

Comment:
I think should probably be changed to `string ...$methodNames` and the same done to `shouldReceive`?